File: app/src/main/java/com/mjr/twaire/app/alleasy/fragment/physic/hydrodynamics/HydrodynamicsFragment.java
package com.mjr.twaire.app.alleasy.fragment.physic.hydrodynamics;

import com.mjr.twaire.app.alleasy.R;
import com.mjr.twaire.app.alleasy.fragment.SubtitlesFragment;
import com.mjr.twaire.app.alleasy.fragment.physic.hydrodynamics.fluidflow.FluidFlowFragment;
import com.mjr.twaire.app.alleasy.model.Subtitle;

public class HydrodynamicsFragment extends SubtitlesFragment {
    @Override
    protected void initializeSubtitles() {
        getSubtitles().add(new Subtitle(getText(R.string.fluid_flow), new FluidFlowFragment()));
    }
    
    @Override
    public String getFragmentTag() {
        return getClass().getPackage().getName() + getClass().getName();
    }
}

File: app/src/main/java/com/mjr/app/alleasy/fragment/physics/hydrodynamics/HydrodynamicsFragment.kt
package com.mjr.app.alleasy.fragment.physics.hydrodynamics

import com.mjr.app.alleasy.R
import com.mjr.app.alleasy.fragment.SubtitlesFragment
class HydrodynamicsFragment : SubtitlesFragment() {
    override fun initializeSubtitles() {
        subtitles!!.add(
            com.mjr.app.alleasy.model.Subtitle(
                getText(R.string.fluid_flow),
                com.mjr.app.alleasy.fragment.physics.hydrodynamics.fluidflow.FluidFlowFragment()
            )
        )
    }
}